packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.controller.js

```diff
@@ -58,6 +58,7 @@ export default /* @ngInject */ function PackInformationCtrl(
     self.servicesStateLink = {
       igmap: IG_MAP_URL.FR,
       travaux: TRAVAUX_URL,
+      guides: GUIDES_URL.FR,
```

Comment:
Can you use userSubsidery to get the guide links? I know as of now its only in FR. It will help us when we support other subsidiaries.

Reply:
@Steffy29 I don't see new commit push yet. Am I missing something regarding @radireddy's comment? (because comment has been marked as _Resolved_, that's why I'm asking)

Reply:
Since ticket received `quality check: ok` label, then I will merge this Pull Request.
@Steffy29 Feel free to provide answer to @radireddy 's question. It will be useful.
Thank you!

Reply:
Sorry, I don't remember I have seen this comment. The answer to this question is : there is not subsidiary for OverTheBox, so I don't use any.